test/Table_Tests/src/Common_Table_Operations/Column_Operations_Spec.enso

```diff
@@ -1091,6 +1107,13 @@ spec setup =
                     vt3.should_be_a (Value_Type.Char ...)
                     vt3.variable_length.should_be_true
 
+        if setup.is_database then
+            t4 = table_builder [["A", ["Alpha", "Bravo", "Charlie", "Delta", "Echo", "Foxtrot"]], ["B", ["A","O","a","E","o","O"]], ["C", [1,2,3,4,5,6]], ["D", ['',Nothing,'',Nothing,'','']]]
+            a = t4.at "A"
```

Comment:
Why construct a table with 4 columns if only `A` is used?

Reply:
Done.